src/__tests__/screen.js

```diff
@@ -1,9 +1,63 @@
 import {renderIntoDocument} from './helpers/test-utils'
 import {screen} from '..'
 
+beforeEach(() => {
+  jest.spyOn(console, 'log').mockImplementation(() => {})
+})
+
+afterEach(() => {
+  console.log.mockRestore()
+})
+
 test('exposes queries that are attached to document.body', async () => {
   renderIntoDocument(`<div>hello world</div>`)
   screen.getByText(/hello world/i)
   await screen.findByText(/hello world/i)
   expect(screen.queryByText(/hello world/i)).not.toBeNull()
 })
+
+test('exposes debug method', () => {
+  renderIntoDocument(
+    `<button>test</button><span>multi-test</span><div>multi-test</div>`,
+  )
+  expect(screen.debug).toBeInstanceOf(Function)
+  // log document
+  screen.debug()
+  expect(console.log).toHaveBeenCalledTimes(1)
+  expect(console.log.mock.calls[0][0]).toMatchInlineSnapshot(`
+    "<body>
+      <button>
+        test
+      </button>
+      <span>
+        multi-test
+      </span>
+      <div>
+        multi-test
+      </div>
+    </body>"
+  `)
+  // log single element
```

Comment:
Let's add a `console.log.mockClear()` between these so that each group of assertions can be more independent from the others.

Reply:
That does make it a-lot more clear. Cool, I appreciate this suggestion.